erpnext/manufacturing/doctype/job_card/job_card.json

```diff
@@ -275,7 +275,7 @@
    "read_only": 1
   },
   {
-   "fetch_from": "work_order.item_name",
+   "fetch_from": "work_order.production_item.item_name",
```

Comment:
This should just be `production_item.item_name`?

fetch_from doesn't work in nested manner. 


Restore branch and reopen PR, so we can get this fix in v12.

Reply:
That's the tricks, I know it should not work, but it is.  So that
production_item.item_name do not work.